bitnami/cilium/values.yaml

```diff
@@ -2069,6 +1953,1437 @@ envoy:
+      ##
+      command: []
+      ## @param hubble.ui.frontend.args Override default Hubble UI frontend container args (useful when using custom images)
+      ##
+      args: []
+      ## @param hubble.ui.frontend.lifecycleHooks for Hubble UI frontend containers to automate configuration before or after startup
+      ##
+      lifecycleHooks: {}
+      ## @param hubble.ui.frontend.extraEnvVars Array with extra environment variables to add to Hubble UI frontend containers
+      ## e.g:
+      ## extraEnvVars:
+      ##   - name: FOO
+      ##     value: "bar"
+      ##
+      extraEnvVars: []
+      ## @param hubble.ui.frontend.extraEnvVarsCM Name of existing ConfigMap containing extra env vars for Hubble UI frontend containers
+      ##
+      extraEnvVarsCM: ""
+      ## @param hubble.ui.frontend.extraEnvVarsSecret Name of existing Secret containing extra env vars for Hubble UI frontend containers
+      ##
+      extraEnvVarsSecret: ""
+      ## @param hubble.ui.frontend.extraVolumeMounts Optionally specify extra list of additional volumeMounts for the Hubble UI frontend containers
+      ##
+      extraVolumeMounts: []
+    ## Backend specific parameters
+    ##
+    backend:
+      ## Bitnami Hubble UI Backend image
+      ## ref: https://hub.docker.com/r/bitnami/hubble-ui-backend/tags/
+      ## @param hubble.ui.backend.image.registry [default: REGISTRY_NAME] Hubble UI Backend image registry
+      ## @param hubble.ui.backend.image.repository [default: REPOSITORY_NAME/hubble-ui-backend] Hubble UI Backend image repository
+      ## @skip hubble.ui.backend.image.tag Hubble UI Backend image tag (immutable tags are recommended)
+      ## @param hubble.ui.backend.image.digest Hubble UI Backend image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag image tag (immutable tags are recommended)
+      ## @param hubble.ui.backend.image.pullPolicy Hubble UI Backend image pull policy
+      ## @param hubble.ui.backend.image.pullSecrets Hubble UI Backend image pull secrets
+      ## @param hubble.ui.backend.image.debug Enable Hubble UI Backend image debug mode
+      ##
+      image:
+        registry: docker.io
+        repository: bitnami/hubble-ui-backend
+        tag: 0.13.1-debian-12-r0
+        digest: ""
+        ## Specify a imagePullPolicy
+        ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+        ## ref: https://kubernetes.io/docs/concepts/containers/images/#pre-pulled-images
+        ##
+        pullPolicy: IfNotPresent
+        ## Optionally specify an array of imagePullSecrets.
+        ## Secrets must be manually created in the namespace.
+        ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+        ## e.g:
+        ## pullSecrets:
+        ##   - myRegistryKeySecretName
+        ##
+        pullSecrets: []
+        debug: false
```

Comment:
Not used

Reply:
addressed at https://github.com/bitnami/charts/pull/27526/commits/6e1f6e34bf685be14e71fc180dee89c349f180ab